### ISSUE TITLE
gzip doesn't work as a parameter at all

### ISSUE THREAD

**Issue by omrihq** [google-github-actions/upload-cloud-storage#18]
### TL;DR
When passing gzip as a parameter in UploadOptions to Storage

**Expected behavior**
I expected the content-encoding not to be gzipped

**Observed behavior**
![Screen Shot 2021-02-04 at 12 29 12 PM](https://user-images.githubusercontent.com/6285455/106952123-87fd4b00-66e5-11eb-942f-92c9083bf910.png)

### Reproduction

I published a forked version of this with `gzip` hardcoded to false. https://github.com/omrihq/upload-cloud-storage/commit/2f3b2496c08da3bf32cee46dd475f2af140477ac

And the encoding is still gzipped. 
**Repository**
Repo: https://github.com/omrihq/upload-cloud-storage/

**Additional information**
I recently merged this expecting to fix this: https://github.com/google-github-actions/upload-cloud-storage/pull/15
I will happily put up another PR when I get to the bottom of it, however this seems not as straightforward as adding the flag originally. 

Thanks @bharathkkb and @averikitsch for being so responsive, I appreciate it!

**Comment by averikitsch** [comment]
@omrihq Did you rebuild the `dist/index.js` file for your test? I just merged the updated `dist/index.js` from your PR. We have a process to automatically generate this. Can you target your workflow at `upload-cloud-storage@main` to test?

**Comment by omrihq** [comment]
@averikitsch I did not build `dest/index.js`, how should I rebuild that? trying again with `google-github-actions/upload-cloud-storage@main`

**Comment by omrihq** [comment]
Hey @averikitsch, I just confirmed that it works. Thanks for bearing with me. Closing this issue!
